src/main/java/seedu/contax/ui/onboarding/OnboardingWindow.java

```diff
@@ -8,10 +9,15 @@
 import javafx.scene.layout.VBox;
 import javafx.stage.Stage;
 import seedu.contax.model.onboarding.OnboardingStep;
+import seedu.contax.model.person.Person;
 import seedu.contax.model.person.UniquePersonList;
 import seedu.contax.ui.PersonListPanel;
 import seedu.contax.ui.UiPart;
 
+/**
+ * The Onboarding Window. Provides a step-by-step guide for first time users
+ * on the basic functions of ContaX.
```

Comment:
Header comment should start with a verb

Reply:
Thanks for the suggestion but will not adjust as it follows the convention of the header comment in MainWindow.